extensions/amp-smartlinks/0.1/linkmate.js

```diff
@@ -163,12 +163,34 @@ export class Linkmate {
    * @private
    */
   getEditInfo_() {
+    const currentUrl = this.extractBaseUrl_(this.ampDoc_.getUrl());
+
     return dict({
       'name': this.rootNode_.title || null,
-      'url': this.ampDoc_.getUrl(),
+      'url': currentUrl,
     });
   }
 
+  /**
+   * Truncates AMP URLs to extract the base url for Narrativ Edit Tracking
+   * Purposes
+   * @param {?string} currentUrl The current URL of the AMP page
+   * @return {?string}
+   * @private
+   */
+  extractBaseUrl_(currentUrl) {
```

Comment:
couldn't we make this a bit more naive and split on `www.` and then just re-assemble the url from there? saves us the while loop.

Reply:
Doy! Good call

Reply:
Could also try using regex. "while" seems a bit inefficient 😅